cpp/benchmarks/groupby/group_scan.cpp

```diff
@@ -69,15 +64,15 @@ void BM_pre_sorted_sum_scan(benchmark::State& state)
 {
   const cudf::size_type column_size{(cudf::size_type)state.range(0)};
 
-  data_profile profile = data_profile_builder().cardinality(0).no_validity().distribution(
+  data_profile profile;
+  profile.set_null_probability(std::nullopt);
+  profile.set_cardinality(0);
+  profile.set_distribution_params<int64_t>(
```

Comment:
Above we collapsed multiple lines of calls to the same profile to a single line. Here we break apart into multiple lines of calls. What is the preferred method?

Reply:
Good catch. Looks like I have a few merge errors.
I'll make another pass to iron these out.

Reply:
I think I get 'em all now. Code should consistently favor one liners (builder).